esque/resources/consumergroup.py

```diff
@@ -28,6 +29,29 @@ def _pykafka_group_coordinator(self) -> pykafka.Broker:
 
         return self._pykafka_group_coordinator_instance
 
+    @property
+    def topics(self):
+        consumer_id = self.id.encode("UTF-8")
+
+        # Get Consumers who already have an offset
+        consumer_offsets = self._unpack_offset_response(
+            self._pykafka_group_coordinator.fetch_consumer_group_offsets(consumer_id, preqs=[])
+        )
+        topic_with_offsets = set(topic.decode("UTF-8") for topic in consumer_offsets.keys())
+
+        topics_with_members = []
```

Comment:
topics_with_members = set(), or else ```TypeError: unsupported operand type(s) for |: 'set' and 'list'``` when there is a struct.error

Reply:
ah god damnit.